docs/docs/cloud/how-tos/test_local_deployment.md

```diff
@@ -0,0 +1,22 @@
+# Test Local Deployment
+
+!!! warning "Browser Compatibility"
+    Viewing the studio page of a local LangGraph deployment does not work in Safari. Use Chrome instead.
+
+## Setup
+
+Make sure you have [setup your app](https://langchain-ai.github.io/langgraph/cloud/deployment/setup/) correctly.
+
+After you have your app setup, [start the API server](https://langchain-ai.github.io/langgraph/cloud/deployment/test_locally/#start-the-api-server) and ensure that it is working properly.
```

Comment:
we should tell them to start it in watch mode to pick up code changes

Reply:
you should put any commands they need to run in this page - they shouldnt have to keep jumping back and forth